e2e/utils/coordinates.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Tab bar coordinates for Maestro E2E tests
+ *
+ * Why this exists:
+ * - Our tab bar can't be interacted with in Maestro tests
+ * - Maestro needs percentage-based coordinates to tap on tab bar items
+ * - Tab bars don't have unique IDs, so we use position-based tapping
+ * - Centralizing coordinates here makes tests more maintainable
+ *
+ * Usage in Maestro tests:
+ * 1. Load: - runScript: { file: e2e/utils/coordinates.js }
+ * 2. Tap:  - tapOn: { point: ${output.tabCoordinates.discoverTab} }
+ */
+
+const tabCoordinates = {
```

Comment:
read the above comment in the file for more info on why this is necessary and how to use it